### ISSUE TITLE
panic: attempt to copy freed scalar in Camelcadedb.pm line 1607

### ISSUE THREAD

**Issue by elw00d** [Camelcade/Perl5-IDEA-Debugger#32]
Got this error sometimes:

```
panic: attempt to copy freed scalar 7f1becbcb880 to 7f1bd2ad0078 at /home/elwood/perl5/lib/perl5/Devel/Camelcadedb.pm line 1607.
```

IDE version:
```
IntelliJ IDEA 2017.2.3
Build #IU-172.3968.16, built on August 22, 2017

JRE: 1.8.0_131-b11 amd64
JVM: Java HotSpot(TM) 64-Bit Server VM by Oracle Corporation
Linux 4.10.0-33-generic
```

Devel::Camelcadedb version:
```
CPAN: Storable loaded ok (v2.41)
Reading '/home/elwood/.cpan/Metadata'
  Database was generated on Wed, 13 Sep 2017 07:17:02 GMT
Devel::Camelcadedb
-------------------------------------------------------------------------
	CPAN: Module::CoreList loaded ok (v3.03)
(no description)
	H/HU/HURRICUP/Devel-Camelcadedb-2017.100.2.tar.gz
	/home/elwood/perl5/lib/perl5/Devel/Camelcadedb.pm
	Installed: 2017.100.2
	CPAN:      2017.100002  up to date
	Alexandr Evstigneev (HURRICUP)
	hurricup@evstigneev.com
```
perl version

```
This is perl 5, version 18, subversion 2 (v5.18.2) built for x86_64-linux-gnu-thread-multi
(with 44 registered patches, see perl -V for more detail)
```

OS where perl is executed:
```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.1 LTS
Release:	14.04
Codename:	trusty
```

OS where IDEA is executed:

```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.3 LTS
Release:	16.04
Codename:	xenial
```

**Comment by elw00d** [comment]
Caused by bug in DBI module: https://rt.cpan.org/Public/Bug/Display.html?id=102791
Fixed in libdbi-perl>=1.635
https://github.com/perl5-dbi/dbi/commit/71802a1e9968c16d00c83b5431d349535abafe12

**Comment by hurricup** [comment]
Thanks!
